### PR TITLE
[SPARK-24711][K8S] Fix tags for integration tests

### PR DESCRIPTION
**Pull request by skonto**
## What changes were proposed in this pull request?

- disables maven surfire plugin to allow tags function properly, doc here: http://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin

## How was this patch tested?

Manually by adding tags.

### CODE REVIEW

**Comment by skonto** [comment]
@ssuchter @foxish @felixcheung pls review.

File: resource-managers/kubernetes/integration-tests/pom.xml [hunk 1]
```diff
@@ -102,6 +103,16 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.22.0</version>
```
Comment (srowen) [inline resource-managers/kubernetes/integration-tests/pom.xml]: I think we declare the version in the parent POM? (or should). OK to update it while you're at it to the latest version.

**Comment by foxish** [comment]
This is good! Same comment that @srowen had on moving the version up to the parent POM and referencing it here.

**Comment by skonto** [comment]
Thanks @foxish will update the PR.

**Comment by skonto** [comment]
@srowen I moved the version to the parent pom. Should I update the version [here](https://github.com/apache/spark/blob/ca8243f30fc6939ee099a9534e3b811d5c64d2cf/pom.xml#L2125) too?

File: pom.xml [hunk 2]
```diff
@@ -114,6 +114,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.version>3.3.9</maven.version>
+    <maven.surefire.plugin.version>2.22.0</maven.surefire.plugin.version>
```
Comment (srowen) [inline pom.xml]: The parent POM here already declares the surefire plugin, but version 2.20.1. That would need to be changed. If you do, you don't need a version in the child POM, nor do you really need a property to control the version and the extra indirection.

Comment (skonto) [reply pom.xml]: Yes but I thought you wanted me to add that version at the parent pom. Otherwise I can simply remove the version from the child pom.

**Comment by skonto** [comment]
@srowen fixed it, ready for final review/merge.

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**